### PR TITLE
test BE stripe payment

### PR DESCRIPTION
**Pull request by tsyaeger**
Tests investment/invest route from request, to creation of investment, to stripe charge.
Adds Faker package.
Creates and associates User, Project, Investment.

### CODE REVIEW

**Comment by skassam21** [comment]
@tsyaeger do you want to see why the tests are not running on circle ci? It may because you need to add enviornmnent variables to circle ci itself here - https://circleci.com/gh/hatchways/team-summer/edit#env-vars

File: server/test/payments.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+require('dotenv').config();
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const faker = require('faker');
+const app = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("/POST Invest", () => {
```
Comment (evankp) [inline server/test/payments.spec.js:10]: Maybe a better description would be "Invest in a project"?

File: server/test/payments.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                        "fundingGoal": faker.random.number()
+                    })
+                    .end((err, res) => {
+                        projectId = res.body._id;
+                        done();
+                    });
+            });
+    });
+
+    it("it should return 200 for successful investment", done => {
```
Comment (evankp) [inline server/test/payments.spec.js:35]: Maybe "Successful investment"?

File: server/test/payments.spec.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                userInfo = res.body;
+                chai.request(app)
+                    .post(`/api/projects`)
+                    .set({ Authorization: `Bearer ${userInfo.token}` })
+                    .send({
+                        "title": faker.lorem.word(),
+                        "description": faker.lorem.words(),
+                        "industry": faker.random.word(),
+                        "location": faker.address.state(),
+                        "fundingGoal": faker.random.number()
```
Comment (evankp) [inline server/test/payments.spec.js:26]: Is there reason why using double quotes here? and even double-quotes where we use single elsewhere

**Comment by evankp** [comment]
I added aws credentials to circleci, just a heads up

**Pull request merged**